fetch.bs

```diff
@@ -2408,6 +2408,10 @@ with a <i>CORS flag</i> and <i>recursive flag</i>, run these steps:
  not <a lt="is local">local</a>, set
  <var>response</var> to a <a>network error</a>.
 
+ <li><p>If |request|'s <a for=request>url</a>'s <a for=url>parser-removed-tab-or-newline flag</a>
+ is set, and |request|'s <a for=request>url</a> <a for=url>path</a> contains a U+003C
+ code point ("<code>&lt;</code>"), then set <var>response</var> to a <a>network error</a>.
```

Comment:
I would be happy with the flag. I don't see how 2 and 3 can work if you want to distinguish between %3C and < on input.